### PR TITLE
Add community newsletter - April 2016

### PR DESCRIPTION
**Pull request by amirfefer**

### CODE REVIEW

File: _posts/2016-04-08-foreman-community-newsletter-april-2016.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Foreman Community Newsletter - April 2016
+author: Amir Fefer
+tags:
+- foreman
+- newsletter
+excerpt: |
+  Foreman 1.11 and 1.10.3 RC2 released. Read on for details of new plugin releases and new recordings to watch!
```
Comment (GregSutcliffe) [inline _posts/2016-04-08-foreman-community-newsletter-april-2016.md]: I don't believe we do RCs for z-stream releases :) What is 1.10.3rc2 meant to refer to?

File: _posts/2016-04-08-foreman-community-newsletter-april-2016.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+On 22th of March Katello 3.0 RC1 has been released. The highlights features include Foreman Remote Execution Integration, OSTree Content Support, Capsule Status Views,  Lazy Sync, Mirror on Sync,  Docker V2 Support and many more
+Check the full [release notes](http://www.katello.org/docs/3.0/release_notes/release_notes.html)
+
+Thanks to everyone who contributed to these releases - code, testing, bug reports, translations, your efforts are very much appreciated!
+
+### New and updated plugins since last March's newsletter
+ - [foreman_ansible_inventory](https://github.com/theforeman/foreman_ansible_inventory) - updated to 0.0.3
+ - [foreman-discovery](https://github.com/theforeman/foreman_discovery) updated to 5.0.2
+ - [hammer-cli-foreman](https://github.com/theforeman/hammer-cli-foreman/tree/0.6.2) updated to 0.6.2
+ - New plugin [smart_proxy_dns_infoblox](https://github.com/sjoeboo/smart_proxy_dns_infoblox) released
```
Comment (GregSutcliffe) [inline _posts/2016-04-08-foreman-community-newsletter-april-2016.md:30]: I believe there's some foreman-cockpit news here too - @dLobatog?

File: _posts/2016-04-08-foreman-community-newsletter-april-2016.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+
+### New media and blogs
+[Community demo](https://youtu.be/P6gX5OHXgbw) on 31th March 2016
+ With most of our developers were busy fixing bugs for the new releases, there wasn't much to demo - instead we sat down with Daniel Lobato Garcia and Eric Helms for an informal meet-the-devs chat!
```
Comment (GregSutcliffe) [inline _posts/2016-04-08-foreman-community-newsletter-april-2016.md:36]: Can you add "Let us know if you'd like to see more of this style of content!" please? I'm keen to hear feedback on this.

File: _posts/2016-04-08-foreman-community-newsletter-april-2016.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+ 
+We continued our [user case study](https://youtu.be/DMWDJuFgyEU) series, and this time with Alex Balk from Outbrain talking about Discovery & Chef. Thanks Alex!
+
```
Comment (GregSutcliffe) [inline _posts/2016-04-08-foreman-community-newsletter-april-2016.md:40]: Could you add https://plus.google.com/b/102496134326414788199/events/cvnrdqab4mhhqvaad4oero18hho as an upcoming event? Thanks!

Comment (GregSutcliffe) [inline _posts/2016-04-08-foreman-community-newsletter-april-2016.md:40]: Oh and Julien Pivotto's talk at OSDC: https://www.netways.de/events_schulungen/osdc/program/julien_pivotto_automating_a_rd_lab_with_foreman_what_can_be_hard/

File: _posts/2016-04-08-foreman-community-newsletter-april-2016.md (hunk 2 above)
Comment (dLobatog) [inline _posts/2016-04-08-foreman-community-newsletter-april-2016.md:30]: Yeah, foreman-cockpit 2.0.1 was released yesterday. I tried to summarize the changes here - https://groups.google.com/forum/#!topic/foreman-users/3hVeZBDkdyU - but summing up: Cockpit component URLs were not 'stabilized' until now. Thus, modern versions of Cockpit (0.78+) use a different set of URLs that the plugin now supports.

File: _posts/2016-04-08-foreman-community-newsletter-april-2016.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+author: Amir Fefer
+tags:
+- foreman
+- newsletter
+excerpt: |
+  Foreman 1.11 and 1.10.3 RC2 released. Read on for details of new plugin releases and new recordings to watch!
+---
+
+### Foreman 1.11 release is out
+Foreman 1.11 was [announced](https://groups.google.com/forum/#!topic/foreman-announce/ZieXOPTC98Q) on 1st of April, and it's not April Fool. The headline features include Improvements to [host and host group](https://youtu.be/PXaVBB88b2Y) parameter editing, [Smart proxy status pages](https://youtu.be/BusFmWMsTbI) enhancements, Smart proxy [DHCP provider]() support, new [host bulk actions](https://youtu.be/xTQuF_KX95Q) and new test buttons for [LDAP and email](https://youtu.be/HP45Eo_JXC0) notifications.
```
Comment (dLobatog) [inline _posts/2016-04-08-foreman-community-newsletter-april-2016.md]: Typo, `April fools`

File: _posts/2016-04-08-foreman-community-newsletter-april-2016.md (hunk 4 above)
Comment (dLobatog) [inline _posts/2016-04-08-foreman-community-newsletter-april-2016.md:40]: There was also a webinar given by Netways in German, https://www.netways.de/webinare/webinare_aktuell/foreman_provisionierungswege/

**Comment by GregSutcliffe** [comment]
Merged as 5ff1c45, thanks @amirfefer!

**Pull request closed without merging**